components/FourPicsTable.js

```diff
@@ -26,7 +26,10 @@ export default function FourPicsTable() {
     <>
       {shirts.map((shirt) => (
         <FourPicsContainer key={shirt._id}>
-          <StyledA href={`/PreviewPage/${searchID}/${shirt.pic1.picSRCSlug}`}>
+          <StyledA
```

Comment:
`StyledLink` would be a better name as this is less confusing (`A` could mean the character `A` or anchor tag. ;))

Reply:
there is a style update upcoming, i try to remember your suggestion!